frontend/src/app/choreography/choreography.component.ts

```diff
@@ -498,4 +500,22 @@ export class ChoreographyComponent {
     }
     this.activeChoreographyItem = item;
   }
+
+  saveToDatabase(): void {
+    const dialogRef = this.dialog.open(SaveChoreographyDialogComponent, {});
+    dialogRef
+      .afterClosed()
+      .pipe(filter((result) => !!result))
+      .subscribe(() => {
+        console.log('hello');
+        this.http
+          .post<Choreography>(`/api/choreographies`, {
+            ...this.choreography,
+            id: 4, // Koreografia -tyypissä on id, mutta olisi parempi että tietokanta loisi sen. Id 4 on testausta varten.
```

Comment:
**suggestion:** change the above to be `Partial<Choreography>`

Reply:
Yeah, that would probably solve the problem of the id. I'll change the type to `Partial<Choreography>`.